docs/MEMORY-SETTINGS.md

```diff
@@ -0,0 +1,495 @@
+# EVE Memory Settings
+
+## TL;DR
+
+```text
+set_global hv_dom0_mem_settings "dom0_mem=1G,max:2G"
+```
+
+sets the soft limit of the `/sys/fs/cgroup/memory/eve/` cgroup to 1G and the
+hard limit to 2G.
+
+```text
+set_global hv_eve_mem_settings "eve_mem=1000M,max:1200M"
+```
+
+sets the soft limit of the `/sys/fs/cgroup/memory/eve/services/` cgroup to
+1000M and the hard limit to 1200M.
+
+```text
+set_global hv_ctrd_mem_settings "ctrd_mem=700M,max:800M"
+```
+
+sets the soft limit of the `/sys/fs/cgroup/memory/eve/containerd/` cgroup to
+700M and the hard limit to 800M.
+
+## Cgroups in EVE
+
+In EVE OS we use cgroups v1 to manage resources.
+
+### What are cgroups
+
+Cgroups are a Linux kernel feature that limits, accounts for, and isolates the
+resource usage of a collection of processes. Cgroups are used to control the
+memory, CPU, and I/O resources that a process can use.
```

Comment:
Should we also explain how cgroup hierarchy works since we are using it? Or maybe just add a link to some official Linux documentation?

Reply:
Will add